src/libraries/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -796,7 +796,7 @@
     <Compile Include="System\Xml\Core\LocalAppContextSwitches.cs" />
     <Compile Include="$(CommonPath)System\CSharpHelpers.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)'=='true'">
+  <ItemGroup Condition="'$(TargetsWindows)'=='true' or '$(TargetsAnyOS)' == 'true'">
```

Comment:
Do we need both conditions? Isn't windows part of any?

Reply:
AnyOS refers to rid-less tfm and is true for that only